scripts/poke.ts

```diff
@@ -0,0 +1,93 @@
+// Common ways of scripted interacting with the Fuse-SDK
+
+// Vendor
+import { ArgumentParser } from "argparse";
+import Joi from "joi";
+import { providers } from "ethers";
+
+// Utilities
+import { validateEnvConfig } from "./utilities/validateConfig";
+
+// SDK
+import { Fuse } from "./sdk/Fuse";
+import { ChainID } from "./utilities/network";
+
+// Validate values from .env file
+const { CHAIN_ID, ETH_RPC_URL, ARBITRUM_RPC_URL, ARBITRUM_RINKEBY_RPC_URL } =
+  validateEnvConfig(
+    ".env",
+    Joi.object({
+      CHAIN_ID: Joi.number().default(1),
+      ETH_RPC_URL: Joi.string().default(""),
+      ARBITRUM_RPC_URL: Joi.string().default(""),
+      ARBITRUM_RINKEBY_RPC_URL: Joi.string().default(""),
+    })
+  );
+
+const RPC_URLS: any = {
+  [ChainID.ARBITRUM]: ARBITRUM_RPC_URL,
+  [ChainID.ARBITRUM_RINKEBY]: ARBITRUM_RINKEBY_RPC_URL,
+  [ChainID.ETHEREUM]: ETH_RPC_URL,
+};
+
+const createParserArguments = (): {
+  flags: any[];
+} => {
+  const parser = new ArgumentParser({ add_help: true });
+
+  parser.add_argument("-f", "--flags", {
+    nargs: "*",
+  });
+
+  return parser.parse_args();
+};
+
+const main = async () => {
+  const parsedArgs = createParserArguments();
+  const provider = new providers.JsonRpcProvider(RPC_URLS[CHAIN_ID]);
+  const fuse = new Fuse(provider, CHAIN_ID);
+
+  // npm run scripts:poke -- -f command arg1 arg2 arg3
```

Comment:
Added the ability to call poke on the CLI to run common functions like `getBorrowableAssetsByIndex` and `getBorrowableAssetsByComptroller`